qa/L0_tf_gpu_io/tf_gpu_io_test.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2023, NVIDIA CORPORATION. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import sys
+
+sys.path.append("../common")
+
+import unittest
+import numpy as np
+import os
+import test_util as tu
+import tritonhttpclient as httpclient
+
+TENSOR_SIZE = 16384
+
+
+class TfGpuIoTest(tu.TestResultCollector):
+
+    def _test_helper(self, model_name, batching_enabled, input_name,
+                     output_name, shape):
+        try:
+            triton_client = httpclient.InferenceServerClient("localhost:8000",
+                                                             verbose=True)
+            inputs = []
+            outputs = []
+            if batching_enabled:
+                shape = [1] + shape
+            inputs.append(httpclient.InferInput(input_name, shape, "FP32"))
+            if batching_enabled:
+                input_data = np.ones(shape=shape, dtype=np.float32)
+            else:
+                input_data = np.ones(shape=shape, dtype=np.float32)
+            inputs[0].set_data_from_numpy(input_data, binary_data=True)
+
+            outputs.append(
+                httpclient.InferRequestedOutput(output_name, binary_data=True))
+            results = triton_client.infer(model_name, inputs, outputs=outputs)
```

Comment:
Can you enhance this test to validate the output results from the inference?

Reply:
Sure, done!